packages/design-system-scripts/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@cmsgov/design-system-scripts",
+  "version": "3.6.0",
```

Comment:
should this be changed to 4.0.0

Reply:
I figured we'd fix all the package versions with lerna when we go to publish or when we merge it into master